assets/app/views/directives/delete-button.html

```diff
@@ -1,7 +1,6 @@
-<div class="actions action-button">
-	<i class="fa fa-trash-o" 
-	  ng-click="$event.stopPropagation(); openDeleteModal();"
-	  data-toggle="tooltip" 
-	  data-placement="left" 
-	  title="Delete"></i>
-</div>
\ No newline at end of file
+<div class="actions">
+  <!-- Avoid whitespace inside the link -->
+  <a href="" ng-click="$event.stopPropagation(); openDeleteModal()" role="button"
+    ><i class="fa fa-trash-o action-button" aria-hidden="true"
+    ></i><span class="sr-only">Delete {{resourceType | humanizeResourceType}} {{resourceName}}</span></a>
```

Comment:
Are we sure that works across screen readers? I can't find definitively one way or the other. The general advice I see is to avoid title.